### ISSUE TITLE
I failed to add the movie

### ISSUE THREAD

**Issue by gr4z** [Waterboy1602/Addarr#19]
I have managed to spend some more time with this and have it installed. Now I try and add a movie to Radarr and get the following: I failed to add the movie

`#Radarr Configuration
radarr:
  server:
    addr: <local IP>
    port : 7878 # Default is 7878
    ssl: false #default false
  auth:
    apikey: <redacted>
    username:
    password:
  search: true
  rootFolder: /radarr
  qualityProfileId: 1

#Telegram Configuration
telegram:
  token: <redacted>
  password: <redacted>

#Transmission Configuration
transmission:
  enable: true #enable the transmission commands
  onlyAdmin: true #transmission command  will only work for admins
  host:
  authentication: true #is transmission protected with a password?
  username:
  password:

#Language
language: en

#Entrypoints
entrypointAuth: auth #auth or a custom entrypoint
entrypointAdd: start #start or a custom entrypoint
entrypointTransmission: transmission #transmission or a custom entrypoint
`
I assume the 'rootfolder' option is when you are accessing the site via /radarr in your URL? What have I missed? 

Thanks

**Comment by Waterboy1602** [comment]
No, `rootFolder` is there to specify to which folder you want Radarr to download your movies. You have the default folder configured in Radarr. So if you want that movies added by Addarr download in a subfolder within the default folder, you should specify this at `rootFolder`. Most of the time you can just leave this empty.

**Comment by gr4z** [comment]
> No, `rootFolder` is there to specify to which folder you want Radarr to download your movies. You have the default folder configured in Radarr. So if you want that movies added by Addarr download in a subfolder within the default folder, you should specify this at `rootFolder`. Most of the time you can just leave this empty.

OK thanks. Have amended, but I still get errors trying to add movies and series.

**Comment by frdmn** [comment]
Please post your errors then.
